.github/workflows/doc-issue.yml

```diff
@@ -0,0 +1,22 @@
+name: Create Issue in docs repo on doc related changes
+
+on:
+  issues:
+    types:
+      - labeled
```

Comment:
Trigger this on "pull_request labeled" is also useful 🥲

Reply:
yes, in github pull request is an issue too. they shares same sequence number and search system.

Reply:
bad, pull request is required explicitly